pkg/agent/ipassigner/ip_assigner_linux.go

```diff
@@ -185,7 +185,7 @@ func (as *assignee) loadIPAddresses() (map[string]*crdv1b1.SubnetInfo, error) {
 				prefixLength, _ := address.Mask.Size()
 				subnetInfo = &crdv1b1.SubnetInfo{
 					PrefixLength: int32(prefixLength),
-					VLAN:         int32(vlanID),
+					VLAN:         uint16(vlanID),
```

Comment:
Why is this change required?

Reply:
I changed this because IPPool used uint16 now and ExternalIPPool use int32. I think we prefer int32 as commented, so I will change IPPool part to int32.